README.md

```diff
@@ -87,13 +87,24 @@ cd snarkOS
 ./build_ubuntu.sh
 ```
 
-Lastly, install `snarkOS`:
+Please ensure ports `4133/tcp` and `3033/tcp` are open on your router and OS firewall.
```

Comment:
why don't we move this right after the `cd snarkOS` step, so we don't duplicate it for the scripts?